agent/util-scripts/pbench-move-results

```diff
@@ -265,9 +279,9 @@ for dir in `/bin/ls -ort -d */ | awk '{print $8}' | grep -v "^tools-" | grep -v
     fi
 
     # Verify the bits copied are good
-    ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo "cd $results_full_path; md5sum --check $pbench_run_name.tar.xz.md5"
+    ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo "cd $results_full_path; tar xf complete-$tarball; rm -f complete-$tarball; md5sum --check $pbench_run_name.tar.xz.md5"
```

Comment:
If we decide to keep this check, then we should implement a server-side script to do the check and invoke it remotely.